### DISCUSSION TITLE
ebusd docker and local configuration files

### DISCUSSION THREAD

**Discussion by UlliSouth** [john30/ebusd#1047] [Q&A]
Hi all,

I just getting started with ebusd running in a docker container. How can I pass my local configuration file? I do not need the files provided by the standard web service. I read the wiki and tried to use the -c option like I did it successfully on the command line. But passing the local .csv  file makes me problems.
Can anyone give me an example how to do that? 

Best
Ulli

**Comment by Koky05** [comment]
Here is part of mine docker-compose file:


> ebusd:
>     image: john30/ebusd:latest
>     container_name: ebusd
>     environment:
>       - EBUSD_CONFIGPATH=/etc/ebusd/ebusd-configuration/ebusd-2.1.x/en
>       - EBUSD_CONFIGLANG=EN
>     volumes:
>       - "/volume1/docker/ebusd:/ebusd"
>       - "/volume1/docker/ebusd/config:/etc/ebusd"
>

**Reply by UlliSouth** [reply]
Thanks a lot!
